src/screens/Account/EmptyStateAccount.js

```diff
@@ -8,134 +8,148 @@ import {
   getMainAccount,
 } from "@ledgerhq/live-common/lib/account";
 import { listTokenTypesForCryptoCurrency } from "@ledgerhq/live-common/lib/currencies";
+import { useRampCatalog } from "@ledgerhq/live-common/lib/platform/providers/RampCatalogProvider";
+import { getAllSupportedCryptoCurrencyIds } from "@ledgerhq/live-common/lib/platform/providers/RampCatalogProvider/helpers";
 import { ScreenName, NavigatorName } from "../../const";
 
 import LText from "../../components/LText";
 import Button from "../../components/Button";
 import Receive from "../../icons/Receive";
 import Exchange from "../../icons/Exchange";
-import { isCurrencySupported } from "../Exchange/coinifyConfig";
 import EmptyStateAccountIllu from "../../images/EmptyStateAccount";
 
 import { withTheme } from "../../colors";
 
-class EmptyStateAccount extends PureComponent<{
+type Props = {
   account: AccountLike,
   parentAccount: ?Account,
   navigation: *,
   colors: *,
-}> {
-  goToReceiveFunds = () => {
-    const { navigation, account, parentAccount } = this.props;
+};
+
+function EmptyStateAccount({
+  account,
+  parentAccount,
+  navigation,
+  colors,
+}: Props) {
+  const mainAccount = getMainAccount(account, parentAccount);
+  const hasSubAccounts = Array.isArray(mainAccount.subAccounts);
+  const isToken =
+    listTokenTypesForCryptoCurrency(mainAccount.currency).length > 0;
+  const currency = getAccountCurrency(account);
+  const rampCatalog = useRampCatalog();
+
+  const canBeBought = useMemo(() => {
```

Comment:
this function can return [boolean, boolean] or boolean, something seems fishy

Reply:
missed this one, fixed